image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -131,6 +131,12 @@ spec:
               resource: limits.cpu
         - name: ROX_CALL_NODE_INVENTORY_ENABLED
           value: {{ if eq ._rox.env.openshift 4 }}"true"{{ else }}"false"{{ end }}
+        - name: ROX_NODE_INDEX_ENABLED
+          value: {{ if eq ._rox.env.openshift 4 }}"true"{{ else }}"false"{{ end }}
+      {{ if eq ._rox.env.openshift 4 }}
```

Comment:
Hi. Please have a look at https://github.com/stackrox/stackrox/blob/master/image/templates/CHANGING_CHARTS.md#whitespace-elimination. For consistency reasons (and to prevent unnecessary whitespaces in the rendered YAML) we should follow the coding style mentioned here.

Reply:
(same below)